README.md

```diff
@@ -13,99 +13,90 @@ This repository contains both the [server](https://github.com/davelopez/galaxy-l
 
 The tags and attributes are suggested based on the [Galaxy.xsd](https://github.com/galaxyproject/galaxy/blob/dev/lib/galaxy/tool_util/xsd/galaxy.xsd) schema. They will appear in the same order that they are declared in the schema, so they can comply with the best practices recommendations defined in the [Galaxy IUC Standards Style Guide](https://galaxy-iuc-standards.readthedocs.io/en/latest/best_practices/tool_xml.html?#coding-style).
 
 
-### Documentation on Hover
+## Documentation on Hover
 
 ![Demo feature hover documentation](../assets/feature.hover.documentation.gif)
 
 The documentation of tags and attributes is retrieved from the [Galaxy.xsd](https://github.com/galaxyproject/galaxy/blob/dev/lib/galaxy/tool_util/xsd/galaxy.xsd) schema.
 >Please note that some elements in the schema are still missing documentation. This will probably be improved over time.
 
 
-### Document validation
+## Document validation
 
 ![Demo feature validation](../assets/feature.validation.png)
 
 The tools are also validated against the [Galaxy.xsd](https://github.com/galaxyproject/galaxy/blob/dev/lib/galaxy/tool_util/xsd/galaxy.xsd) schema.
 
 
-### Document auto-formatting
+## Document auto-formatting
 
 ![Demo feature auto-formatting](../assets/feature.autoformat.gif)
 
 When the tool file is saved it gets auto-formatted to maintain a consistent format with the [Galaxy IUC Standards Style Guide](https://galaxy-iuc-standards.readthedocs.io/en/latest/best_practices/tool_xml.html?#coding-style).
 
-### Tag auto-closing
+## Tag auto-closing
 
 ![Demo feature auto-close tags](../assets/autoCloseTag.gif)
 
 Whenever you write a closing ``>`` the corresponding closing tag will be inserted. You can also type ``/`` in an open tag to close it.
 
 
-### Snippets
+## Snippets
 
 ![Demo snippets](../assets/snippets.gif)
 
 Snippets can be really helpful to speed up your tool wrapper development. They allow to quickly create common blocks and let you enter just the important information by pressing ``tab`` and navigating to the next available value.
 >If you want to add more snippets check the [guide](./docs/CONTRIBUTING.md#adding-snippets) in the contribution guidelines.
 
-### Embedded syntax highlighting
+## Embedded syntax highlighting
 
 ![Demo feature embedded syntax highlighting](../assets/feature.embedded.syntax.png)
 
 Basic support for `Cheetah` and `reStructuredText` syntax highlighting inside the `<command>`, `<configfile>` and `<help>` tags. The embedded code should be inside a `CDATA` block.
 
-### Auto-generate tests
+## Auto-generate tests
 
 ![Demo feature auto-generate tests](../assets/feature.generate.tests.gif)
 
 After you define the `<inputs>` and `<outputs>` of the tool, you can press `Ctrl+Alt+t` (or `Cmd+Alt+t` in Mac) to create a `<tests>` section with a basic structure and some test cases. This is especially useful when using conditionals and other nested parameters since you can get right aways most of the bolerplate XML.
```

Comment:
```suggestion
After you define the `<inputs>` and `<outputs>` of the tool, you can press `Ctrl+Alt+t` (or `Cmd+Alt+t` in Mac) to create a `<tests>` section with a basic structure and some test cases. This is especially useful when using conditionals and other nested parameters since you can get right away most of the boilerplate XML.
```